content/post/release-of-1.5.0.md

```diff
@@ -0,0 +1,303 @@
+not as big as a feature. It might not be instantly clear to everyone who reads
+it, but you can help them understand through the use of the new descriptions!
+
+_Thanks to [**@lafriks**](https://github.com/lafriks)_
+
+## Total Tracked Time ([#3341](https://github.com/go-gitea/gitea/pull/3341))
+
+![Demo on a milestone](/demos/3341/1.png)
+
+You can now track the total amount of time you spent on a single issue - or even
+an entire milestone.
+
+_Thanks to [**@JonasFranzDEV**](https://github.com/JonasFranzDEV)_
+
+## Other changes
+
+* You can now specify a second whitelist for protected branches, allowing you
+  to select users who are able to merge PRs.
+  ([#3689](https://github.com/go-gitea/gitea/pull/3689))
+* We did some optimisations on the repository search feature - users report up
+  to a 3x reduction in disk usage by the indexer.
+  ([#3452](https://github.com/go-gitea/gitea/pull/3452))
+* Power to webhooks! We added support for delete, fork, issues, issue\_comment,
+  and release webhooks, which have long been a requested feature ever since
+  webhooks first came to Gogs.
+  ([#3929](https://github.com/go-gitea/gitea/pull/3929))
+* Some changes were made to how Gitea handles messages with custom markup, such
+  as commit messages and issue comments. Mentions, emails, links and so on
+  should now be correctly handled.
+  ([#3354](https://github.com/go-gitea/gitea/pull/3354))
+* If your team is used to placing issue references in square brackes
+  (ie. `[JIRA-123]`), we now correctly parse that!
+  ([#3408](https://github.com/go-gitea/gitea/pull/3408))
+* From the admin panel, you can now run `git fsck` (health check) on all your
+  repositories, as well as disable it entirely if desired.
+  ([#3606](https://github.com/go-gitea/gitea/pull/3606),
+  [#3607](https://github.com/go-gitea/gitea/pull/3607))
+* We added some new features to Gitea's API, such as issue search and
+  attachments. ([#3478](https://github.com/go-gitea/gitea/pull/3478),
+  [#3612](https://github.com/go-gitea/gitea/pull/3612))
+* Symlinks in a repository are now marked by a distinctive icon.
+  ([#3826](https://github.com/go-gitea/gitea/pull/3826))
+* We'll remember your preferred language so that you don't have to change it for
+  each browser that you use.
+  ([#3875](https://github.com/go-gitea/gitea/pull/3875))
+* If you want to disable time tracking entirely, you can now do so from the app
+  settings. ([#3719](https://github.com/go-gitea/gitea/pull/3719))
+* Various changes to improve consistency and grammar in the English
+  localisation.
+  ([Various](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+author%3Abugreport0+is%3Aclosed+milestone%3A1.5.0))
+* You can now sort repos in Explore and the admin panel by stars or forks.
+  ([#3969](https://github.com/go-gitea/gitea/pull/3969))
+* Starting from 1.5.0, we'll sign all our releases with our
+  [GPG Key,](http://pool.sks-keyservers.net/pks/lookup?op=get&hash=on&fingerprint=on&search=0x2D9AE806EC1592E2)
```

Comment:
http://pool.sks-keyservers.net/pks/lookup?op=get&hash=on&fingerprint=on&search=0x2D9AE806EC1592E2 returns 404 for me

Reply:
Seems like it's fixed now, I get a GPG key when I go to that URL.